wear/build.gradle

```diff
@@ -157,26 +157,27 @@ allprojects {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    compileOnly 'com.google.android.wearable:wearable:2.0.1'
-    implementation 'com.google.android.support:wearable:2.0.1'
-    implementation 'com.android.support:appcompat-v7:23.1.1'
+    compileOnly 'com.google.android.wearable:wearable:2.5.0'
+    implementation 'com.google.android.support:wearable:2.5.0'
+    implementation 'com.android.support:appcompat-v7:26.1.0'
     // implementation 'com.google.android.support:wearable:1.4.0'
     //implementation 'com.google.android.gms:play-services-wearable:9.4.0'
-    implementation 'com.google.android.gms:play-services-wearable:10.2.1'
+    // FIX upgrage to play-services-wearable:17.0.0 requires upgrade to AndroidX libraries, https://developers.google.com/android/guides/releases#june_17_2019
+    implementation 'com.google.android.gms:play-services-wearable:16.0.1'
```

Comment:
Different versions of play services work in very different ways and may not be backwards compatible. How much testing has been done with the bi-directional syncing with Android wear and this version?

Also I'm pretty sure this will break rooted SWR50s but these may be far less common these days.

Reply:
Does this make the `play-services-wearable` library a different version between app and wear?

Reply:
I am testing this with Sony Smartwatch 3 which was not syncing data to watchface for me and with this pull request it does. I am not able to test it as collector.

My version for testing build with this pull request is here http://bit.do/xdrip-k2s (https://gist.github.com/k2s/aa7928197d8ea508135bc20bb9bdd664) if somebody has the possibility to test it as collector.

You are right I didn't upgraded the play-services-wearable in app, because there are more play-services libraries at version 10.2.1. Maybe they should be and also firebase-messaging be upgraded too ?

It compiles also with the older version of play-services-wearable in wear. I have to test if it will not break syncing with my watch, becasue that was the reason why I upgraded it.

Reply:
> Also I'm pretty sure this will break rooted SWR50s but these may be far less common these days.

I'm using one of those quite happily, readily recommended it to others in the past, and am supporting at least one other happy user in my vicinity with that setup. So I'd be sad to see support for the SW50 dropped.